plugins/ruby-foreman-rescue/debian/install

```diff
@@ -0,0 +1,2 @@
+foreman_rescue.rb              usr/share/foreman/bundler.d
+cache                          usr/share/foreman/vendor
```

Comment:
Looks like this is failing the packaging: 
```
17:38:13 cp cache/* /usr/share/foreman/vendor/cache/
17:38:13 cp: cannot create regular file '/usr/share/foreman/vendor/cache/': No such file or directory
```

Reply:
I obviously did something wrong. I will get back to you ASAP

Reply:
no rush :)